apps/src/containedLevels.js

```diff
@@ -18,16 +17,16 @@ let callOnPostCompletion = null;
  */
 export function getContainedLevelResultInfo() {
   const containedResult = codeStudioLevels.getContainedLevelResult();
-  const levelResult = containedResult.result;
-  const testResult = valueOr(containedResult.testResult,
-    levelResult.result ? TestResults.ALL_PASS : TestResults.GENERIC_FAIL);
   return {
     app: containedResult.app,
     level: containedResult.id,
     callback: containedResult.callback,
-    result: levelResult.result,
-    testResult: testResult,
-    program: levelResult.response,
+    // We only care whether they've submitted or not, and in many cases don't even
+    // know as the client if the submission was correct or not, as we're often
+    // not provided correct answers (i.e. in multis).
+    result: true,
+    testResult: TestResults.CONTAINED_LEVEL_RESULT,
```

Comment:
It should be the case that it doesn't get called until they've answered (this gets called when they click run, which is disabled until they've selected an answer in case of multi, or entered some text in the case of free response)